rotkehlchen/globaldb/cache.py

```diff
@@ -122,3 +122,12 @@ def globaldb_delete_general_cache(
         write_cursor.execute('DELETE FROM general_cache WHERE key=?', (cache_key,))
     else:
         write_cursor.execute('DELETE FROM general_cache WHERE key=? AND value=?', (cache_key, value))  # noqa: E501
+
+
+def globaldb_delete_general_cache_like(
+        write_cursor: DBCursor,
+        key_parts: Iterable[Union[str, GeneralCacheType]],
+) -> None:
+    """Function to delete globaldb cache. Deletes all the values that start with key."""
+    cache_key = _compute_cache_key(key_parts)  # note: cache key should not contain `.` or `%`
```

Comment:
This looks like a bug waiting to happen. Can you add an assertion here so if that happens at least we know?

Reply:
yes, will add